docs/_guide/templates.md

```diff
@@ -8,21 +8,35 @@ slug: templates
 * ToC
 {:toc}
 
+Add a template to your component to define internal DOM to implement your component. 
+
+LitElement uses
```

Comment:
How about flipping the order, to focus on the user benefit, rather than the implementation? Ie: 

To encapsulate the templated DOM and provide scoped styling, LitElement uses [shadow DOM](...). That way, you can add styles ...

Maybe we can even point out here, and in other places, why this is better and different than other approaches. Basically, it's a web standard way. No verbose BEM or extra build step required to munge your css class names to scope them.

Reply:
I've re-written this to flip the order, and also expand a little bit: focusing on just one benefit (style encapsulation) confuses the issue.

Need to decide whether this is getting a bit too long and needs to be moved elsewhere.